src/components/Search/Result/RecordDetails.js

```diff
@@ -1,43 +1,46 @@
 // @flow
 import * as React from 'react';
 import { connect } from 'react-redux';
-import { AccordionSet, Col, Row, FilterAccordionHeader, Accordion } from '@folio/stripes/components';
+import { AccordionSet, Col, Row, FilterAccordionHeader, Accordion, MultiColumnList } from '@folio/stripes/components';
 import InventoryPluggableButton from '../Button/Inventory';
 import type { Props } from '../../../flow/types.js.flow';
 import AssociatedBib from './AssociatedBib';
 import { ACTION } from '../../../redux/actions';
-
 import { mapFields } from '../Utils/SearchUtils';
 import { FixedFields } from '../../Cataloguing/Models/model';
 import { SUBFIELD_CHARACTER } from '../../Cataloguing/Utils/MarcConstant';
-
+import { SEARCH_SEGMENT } from '../../../config/constants';
 import style from '../Style/index.css';
+import {
+  authDetailsColumnMapper,
+  authDetailsRenderColumn,
+  authDetailsResultFormatter,
+} from '../../../shared/utils/Formatter';
 
 type P = Props & {
   items: Array<any>,
 }
 
 class RecordDetails extends React.Component<P, {}> {
-  constructor(props: P) {
-    super(props);
-    const id = props.detailPaneMeta.meta['001'];
-    let mergedResults;
-    let detailSelected;
-    if (props.data.search.dataOld !== undefined) {
-      mergedResults = [...props.data.search.bibliographicResults, ...props.data.search.oldBibArray];
-      detailSelected = mergedResults.filter(item => id === item.data.fields[0]['001']);
-    } else {
-      detailSelected = props.data.search.bibliographicResults.filter(item => id === item.data.fields[0]['001']);
-    }
+  render() {
+    const { store: { dispatch }, detailPaneMeta, checkDetailsInRow, translate, checkDetailsBibRec, data: { search: { segment } } } = this.props;
+    const list = [
```

Comment:
Why is there sample data in the component?

Reply:
It was just sample data to show on a table and to verify it worked fine. I have make a new commit with the code cleaned